adapters/handlers/graphql/local/get/class_builder_fields.go

```diff
@@ -596,6 +598,8 @@ func fieldNameIsOfObjectButNonReferenceType(field string) bool {
 		"valid", "countryCode", "defaultCountry":
 		// must be a phone number
 		return true
+	// case "_additional":
+	// 	return false
```

Comment:
is this part needed? or can it be removed?

Reply:
removed